lib/Models/LocationMarkerUtils.ts

```diff
@@ -91,7 +93,24 @@ export function removeMarker(terria: Terria) {
 }
 
 /** Determines whether the location marker is visible previously added in {@link #addMarker}. */
-export function isMarkerVisible(terria: Terria) {
+export function isMarkerVisible(terria: Terria): boolean {
   const catalogItem = terria.getModelById(CzmlCatalogItem, MARKER_UNIQUE_ID);
   return catalogItem !== undefined && terria.overlays.contains(catalogItem);
 }
+
+export function getMarkerLocation(terria: Terria): LatLonHeight | undefined {
+  const catalogItem = terria.getModelById(CzmlCatalogItem, MARKER_UNIQUE_ID);
+  if (catalogItem === undefined || !terria.overlays.contains(catalogItem)) {
+    return;
+  }
+  const marker = catalogItem.czmlData[1];
+  // @ts-ignore
```

Comment:
I'm not a fan of `// @ts-ignore`. It can be broken by changes in indenting because prettier could reflow lines, either ignoring types for more or less code than intended and it's less clear exactly where it's needed than casting `((o as unknown) as MyType)` or using `<any>o`